docs/get-started.md

```diff
@@ -1,170 +1,64 @@
-Getting Started
-===============
-
-In order to consume these components you will require a system that
-utilised a bundler (such as [Webpack]) that can process imported assets
-such as images, fonts and [Sass] files. [Create React App] may be able to
-do this but does not provide Server-Side Rendering (SSR). You will also
-need to ensure that you provide an instance of [react-router].
-
-As such, we suggest that you use our specially designed tech stack for
-this purpose. You can start a brand new project or prototype using the
-following steps.
+Prototyping
+===========
 
+This guide explains how to create prototypes using the Home Office Design System.
 
 ### 1. Set up your repository
 
-You should first set up a blank code repository in the provider of your
-choice. For the purposes of this document I will assume [GitHub] as we
-provide extra CI support through [GitHub Actions].
+Set up a blank code repository in [GitHub] as
+extra CI support is provided through [GitHub Actions].
```

Comment:
For a prototype they might not need CI.

Although that is useful if they want to deploy to Netlify.

Reply:
I'll leave the content as it is for now and perhaps we can review this later as we gather more information.